src/hardware/openbench-logic-sniffer/protocol.h

```diff
@@ -65,6 +65,10 @@
 #define FLAG_FILTER                (1 << 1)
 #define FLAG_DEMUX                 (1 << 0)
 
+/* Capture context magic numbers */
+#define CTX_NO_TRIGGER (-1)
```

Comment:
Why CTX? Maybe OLS_NO_TRIGGER?

Reply:
Good point, it's not part of the context itself. Renamed to `OLS_NO_TRIGGER`.